generators/app/templates/ext-notebook/src/notebook.ts

```diff
@@ -4,42 +4,11 @@ import * as vscode from 'vscode';
 import * as azdata from 'azdata';
 import * as path from 'path';
 import * as fs from 'fs';
-import * as os from 'os';
-
-// This function looks in the user's default extensions folder for Azure Data Studio
-// to find this extension and its packaged files. If it encounters an error, a
-// message will appear in an error window.
-const processNotebooks = () => {
-    const rootExtensionsFolder = path.normalize(path.join(os.homedir(), '.azuredatastudio', 'extensions'))
-    let notebookNames: Array<string> = [];
-
-    let subExtensionFolder = getFolderContent(rootExtensionsFolder);
-
-    subExtensionFolder.forEach(folderName => {
-        findCorrectFolder(folderName, rootExtensionsFolder, notebookNames);
-    });
-    return notebookNames;
-}
-
-// This function is called by processNotebooks to find the correct folder that contains
-// this extension. If it is found, then it opens up the book in Azure Data Studio's
-// native notebook viewlet.
-const findCorrectFolder = (folderName: string, rootExtensionsFolder: string, notebookNames: Array<string>) => {
-    let folderExt = path.basename(folderName).toLowerCase();
-
-    if (folderExt.indexOf(('<%= publisherName%>.<%= name%>').toLowerCase()) > -1) {
-        let fullFolderPath = path.join(rootExtensionsFolder, folderName, 'content');
-        try {
-            extractNotebooksFromFolder(fullFolderPath, notebookNames);
-        } catch (e) {
-            vscode.window.showErrorMessage("Unable to access " + fullFolderPath + ": " + e.message);
-        }
-    }
-}
 
 // Each notebook or markdown file packaged with your extension will be found through this
 // function so that they can be individually opened through the `showNotebookDocument` command.
-const extractNotebooksFromFolder = (fullFolderPath: string, notebookNames: Array<string>) => {
+const extractNotebooksFromFolder = (fullFolderPath: string) => {
+    let notebookNames: Array<string> = [];
     const files = getFolderContent(fullFolderPath);
```

Comment:
if i read this correctly, you only handle the files in the top level folder without recursively going to the sub folders, by design?